test/coordtest/linearizability.ct

```diff
@@ -0,0 +1,107 @@
+inc-timestamp
```

Comment:
why did we need this?

Reply:
Good catch. I tried to remove this but then it panic'd at https://github.com/MaterializeInc/materialize/blob/d02b8486beda22c5af8cf8cec3fed5ea6508d9ed/src/dataflow/src/source/timestamp.rs#L80 so apparently it's necessary.